File: ObjCSample/ObjCSample/Models/MenuItem.m
//
//  MenuItem.m
//  ObjCSample
//
//  Created by Julien GOMEZ on 16/10/18.
//  Copyright (c) 2018 Smart AdServer. All rights reserved.
//

#import "MenuItem.h"
#import "BannerViewController.h"

@implementation MenuItem

- (id)initWithTitle:(NSString *)title segueIdentifier:(NSString *)segueIdentifier {
	self = [super init];
	
	if (self) {
		self.title = title;
        self.segueIdentifier = segueIdentifier;
	}
	return self;
}

@end

File: SwiftSample/SwiftSample/Models/MenuItem.swift
//
//  MenuItem.swift
//  SwiftSample
//
//  Created by Julien Gomez on 25/10/2018.
//  Copyright © 2018 Smart AdServer. All rights reserved.
//

import Foundation

/*
 Represents an entry in the Main menu.
 */
class MenuItem: NSObject {
	
	let title: String
	let segueIdentifier: String
	
	init(title: String, segueIdentifier: String) {
		self.title = title
		self.segueIdentifier = segueIdentifier
		super.init()
	}
	
}
